test/regression_tests/shallow_water_bickley_jet_regression.jl

```diff
@@ -40,7 +41,7 @@ function run_shallow_water_regression(arch, formulation; regenerate_data = false
     end
 
     stop_iteration = 20
-    Δt = 1e-2
+    Δt = 5e-3
```

Comment:
```suggestion
    Δt = 1e-6
```

Reply:
if 5e-3 is not enough I'll reduce it to 1e-6

Reply:
Word. No need to cut it close at all here